pool/cqrs_command/filter_command.ml

```diff
@@ -36,7 +36,7 @@ module Create : sig
     :  (string * string list) list
     -> (t, Pool_common.Message.error) result
 
-  val effects : Filter.t -> Guard.Authorizer.effect list
+  val effects : Guard.Authorizer.effect list
```

Comment:
use `include Common.CommandSig with type t = Filter.Title.t` for line 25, and this line `val effects: ...` can be removed (part of CommandSig)

Reply:
remove `val effects : Guard.Authorizer.effect list`